src/wrapped_radix_engine_toolkit/low_level/transaction_intent.rs

```diff
@@ -4,29 +4,32 @@ use crate::prelude::*;
 #[debug("header:\n{:?}\n\nmessage:\n{:?}\n\nmanifest:\n{}\n\n", self.header, self.message, self.manifest.instructions_string())]
 pub struct TransactionIntent {
     pub header: TransactionHeader,
-    pub manifest: TransactionManifest,
+    manifest: TransactionManifest,
```

Comment:
Hide constructor so that we ensure we always go via `new` which now is a failable constructor, performing `compile_intent_with ` - checking SBOR depth.